deploy/examples/create-external-cluster-resources.py

```diff
@@ -1139,66 +1085,57 @@ def gen_json_out(self):
 
         return json.dumps(json_out)+LINESEP
 
-    def upgrade_user_permissions(self):
+    def upgrade_users_permissions(self):
+        users = ["client.csi-cephfs-node","client.csi-cephfs-provisioner","client.csi-rbd-node","client.csi-rbd-provisioner"]
+        if self.run_as_user != "":
```

Comment:
Brought this check here for checking if any restricted user is passed,
Suggested by @aruniiird 
Thanks!

Reply:
Can you do a small additional check like the user provided (through `--run-as-user`) is not already there in the `users` list

```python
if self.run_as_user and self.run_as_user not in users:
    users.append(self.run_as_user)
```

Reply:
DOne